io_scene_niftools/ui/operators/nif_import.py

```diff
@@ -161,6 +161,13 @@ def draw(self, context):
         layout.prop(operator, "send_geoms_to_bind_pos")
         layout.prop(operator, "apply_skin_deformation")
 
+        layout.prop(operator, "override_armature_orientation")
+        row_ax_fwd = layout.row()
+        row_ax_fwd.prop(operator, "armature_axis_forward")
+        row_ax_fwd.enabled = operator.override_armature_orientation #Grey out unless requesting override
```

Comment:
This should be moved to the poll function.

Reply:
I'm confused on how to move that to the poll - I've never interacted with layouts in the poll calls?

Reply:
This might be fine for now. 
Will need to look further as to see if there is a preferred way to do it, possibly with a subpanel

Reply:
Had a browse around some other ui panels, looks like its fine 👍